src/accordion/accordion.spec.ts

```diff
@@ -552,25 +555,37 @@ describe('ngb-accordion', () => {
     expect(panelContents.length).toBe(3);
   });
 
-  it('should emit panel change event when toggling panels', () => {
+  it('should emit panel events when toggling panels', () => {
     const fixture = TestBed.createComponent(TestComponent);
     fixture.detectChanges();
 
     fixture.componentInstance.changeCallback = () => {};
```

Comment:
please remove all these:

```ts
fuxture.componentInstance.xxx = () => {}
```

They're already declared in the `TestComponent`, no need to re-declare them